frontend/src/views/c-zoom.vue

```diff
@@ -478,6 +541,7 @@ export default defineComponent({
   /* Commit Message Body in Zoom Tab */
   .commit-message {
     border: 1px solid transparent;
+    overflow: hidden;
```

Comment:
Thank you for pointing this out. I will try to find a fix that still shows the tooltip.

Previously, some diffstats were spilling over onto the next commit, causing the contribution bar to "stick" to the commit title of the next commit.

![image](https://github.com/reposense/RepoSense/assets/97350362/718d5d8d-204b-4966-b5f4-2ca572f6cfe7)

Reply:
Think this is due to the `float: left` property on the contribution bars which makes the parent container have a height of 0, resulting in the elements below not being pushed down.

From searching online the solutions seem to be either using `overflow: hidden` as you've tried (but results in our tooltip being hidden), or some CSS tricks (clearfix?)

Alternatively, it might be worth trying to remove the `float: left` and find an alternative method to position the bars, so the bars will actually have the correct height. Since the bars are just divs with a `width: xx%` property, maybe putting them in a flex container with flex wrap will work?